distribution/deps.bzl

```diff
@@ -6,8 +6,18 @@
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```

Comment:
I'll fix it once the bazel configuration is merged: https://github.com/vaticle/bazel-distribution/pull/406